src/open_inwoner/components/templates/components/Form/Form.html

```diff
@@ -9,6 +9,12 @@
     {% if data_confirm_title %}data-confirm-title="{{ data_confirm_title }}"{% endif %}
     {% if data_confirm_cancel %}data-confirm-cancel="{{ data_confirm_cancel }}"{% endif %}
     {% if data_confirm_default %}data-confirm-default="{{ data_confirm_default }}"{% endif %}
+    {% if async_selector %}
+        hx-{{ method|lower }}="{{ action|default:"./" }}"
+        hx-select="{{ async_selector }}"
+        hx-target="{{ async_selector }}"
```

Comment:
Should we also add polling for messages? According to the docs it can be done with 1 attribute
@alextreme @svenvandescheur what do you think?

Reply:
https://htmx.org/docs/#polling